tmt-app/src/utils/constants.js

```diff
@@ -6,112 +6,209 @@ export const CAREERS_LIST = {
+    majors: [
+      "Economics & Business",
+      "Languages",
+      "Physical Science & Engineering",
+      "Social & Political Science",
+    ],
     key: "law-legal",
   },
   marketing: {
     id: 11,
     name: "Marketing",
     desc: "Marketing is the process of identifying, anticipating, and satisfying customer needs and wants through the creation, promotion, and distribution of products and services. It involves conducting market research, developing marketing strategies and campaigns, and using various tactics, such as advertising and social media, to reach and engage with potential customers. Marketing professionals are responsible for promoting products and services to target audiences in order to increase sales and grow a company's market share",
     link: "https://www.zippia.com/marketing-manager-jobs/",
+    skills: [
+      "Copywriting",
+      "Search Engine Optimization",
+      "Customer Relationship Management",
+    ],
+    majors: [
+      "Economics & Business",
+      "Humanities & Cultures",
+      "Languages",
+      "Psychology & Neuroscience",
+      "Social & Political Science",
+    ],
     key: "marketing",
   },
   "product-management": {
     id: 12,
     name: "Product Management",
     desc: "Product management is the process of defining, developing, and bringing a product to market. It involves working with a team of designers, engineers, and other stakeholders to identify the needs of the target audience, develop a product roadmap, and create a plan to bring the product to market. Product managers are responsible for overseeing the entire product lifecycle, from conception to launch and beyond, and for making strategic decisions about the direction of the product",
     link: "https://www.zippia.com/product-manager-jobs/demographics/",
+    skills: ["Market Research", "Product Vision", "Roadmapping"],
+    majors: [
+      "Computer Science & Math",
+      "Physical Science & Engineering",
+      "Visual Arts & Design",
+    ],
     key: "product-management",
   },
 
-  "project-project-management": {
+  "project-program-management": {
     id: 13,
     name: "Project/Program Management",
     desc: "Project management is the process of planning, organizing, and overseeing the work of a team to achieve specific goals and meet deadlines. It involves establishing a project plan, identifying and acquiring the necessary resources, and managing the budget, scope, and risks of the project. Project managers are responsible for ensuring that the project stays on track and is completed successfully",
     link: "https://www.zippia.com/program-manager-jobs/",
+    skills: ["Stakeholder Engagement", "Risk Management", "Change Management"],
+    majors: [
+      "Computer Science & Math",
+      "Economics & Business",
+      "Education",
+      "Humanities & Cultures",
+      "Physical Science & Engineering",
+      "Psychology & Neuroscience",
+    ],
     key: "project-program-management",
   },
```

Comment:
Content decided to group these?

Cc: @victoriatran17

Reply:
Yes @jwu910

Reply:
Oops wrong team